android/src/main/java/io/parity/signer/bottomsheets/SignatureReady.kt

```diff
@@ -64,7 +64,7 @@ fun SignatureReady(
 			BigButton(
 				text = "Done",
 				action = {
-					signerDataModel.pushButton(Action.GO_BACK, "", "")
+					signerDataModel.navigate(Action.GO_BACK, "", "")
```

Comment:
They are default. I just renamed function and didn't remove arguments, but they can be removed and result will be the same.

Reply:
I guess it's better to remove then if there are default values already for it 🙌🏻

Reply:
There are bunch of things need to be done. And I would like to invest time to fix completely broken navigation, that doesn't support non-rust driven views at the moment, rather than going through the code and removing values, explicitly showing us in call side of which is passed.
And even before that I would like to invest time in passing navigator to a view as dependency rathen than calling static method from views. Just not in this PR.